openapi/nexus.json

```diff
@@ -4015,6 +4015,33 @@
         }
       }
     },
+    "/v1/system/hardware/sleds-uninitialized": {
+      "get": {
+        "tags": [
+          "system/hardware"
+        ],
+        "summary": "List uninitialized sleds in a given rack",
+        "operationId": "sled_list_uninitialized",
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/UninitializedSledResultsPage"
+                }
+              }
+            }
+          },
+          "4XX": {
+            "$ref": "#/components/responses/Error"
+          },
+          "5XX": {
+            "$ref": "#/components/responses/Error"
+          }
+        }
```

Comment:
we'd expect to see "x-dropshot-paginated" in here

Reply:
`x-dropshot-pagination` shows up as of https://github.com/oxidecomputer/omicron/pull/4720